### DISCUSSION TITLE
Should the output of both backends be identical (e.g., for SD1.5)?

### DISCUSSION THREAD

**Discussion by anothertal3** [vladmandic/sdnext#3207] [Q&A] [answered]
Hi everyone,

after the last update I decided to switch over from the "original" backend to the new one ("diffusers"). 

To do so, I did a fresh installation of SD.Next. There are now two SD.Next instances on my computer:
- A, which is the current version with a few adjusted settings and the original backend
- B, which is a clean installation of the current version using default settings and the diffusers backend

I then tried to reproduce the output of A with B and found that I am unable to do so. I also copied settings for pictures on civitai and attempted to recreate them in B. The results were mostly quite different.

Before diving too deep into finer settings and looking for a potentially non-existing issue: what is the expected behavior for my scenario? Should the output be comparable or is this behavior expected?

Thank you!

**Comment by vladmandic** [comment, marked as answer]
output for both backends is not expected to be identical, just like output of different applications like invoke vs a1111 is not expected to be identical. differences between original backend and diffusers backend are just too great.

**Reply by anothertal3** [reply]
Understood, thank you for the explanation
